### PR TITLE
Update landing page for osdc.github.io

### PR DESCRIPTION
**Pull request by saranshbarua**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    <link rel="stylesheet" href="styles/style.css">
+  </head>
+  <body>
+    <div class="main">
+      <div class="gradientbox">
+          <div class="heading">OSDC</div>
+      </div>
+      <div class="subheading">The Open Source Community of JIIT</div>
+    </div>
+    <script type="text/javascript" src="scripts/script.js"></script>
```
Comment (526avijitgupta) [inline index.html]: This file seems to be empty. Included accidentally or yet to be updated ?

File: styles/style.css [hunk 2]
```diff
@@ -0,0 +1,187 @@
+body
```
Comment (526avijitgupta) [inline styles/style.css]: Formatting changes:

* Please follow the opening brace in the same line. For example:

```css
.class {
  ...
}
```

* Please add space after a semicolon everywhere. For example it should be `margin: 0` and not `margin:0`.

* Preferably leave one line gap after each rule, for clarity.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>osdc.github.io</title>
+    <link href="https://fonts.googleapis.com/css?family=Assistant:200|Monoton" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Roboto:300" rel="stylesheet">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="stylesheet" href="styles/style.css">
+  </head>
+  <body>
+    <div class="main">
```
Comment (526avijitgupta) [inline index.html]: Too many "divs" - redundant and not SEO friendly. Please try to use HTML5 tags like `<main>`, `<header>`, `<section>` wherever possible.
For example, I guess you could replace `<div class="main">` by simply `<main>` here.

**Comment by saranshbarua** [comment]
[https://saranshbarua.github.io/osdc.github.io/](url)

**Comment by saranshbarua** [comment]
[OSDC landing page](https://saranshbarua.github.io/osdc.github.io/)

File: index.html (hunk 1 above)
Comment (saranshbarua) [reply index.html]: It is yet to be updated.

File: styles/style.css (hunk 2 above)
Comment (saranshbarua) [reply styles/style.css]: Will surely look into these in further code.Also will try to change wherever possible.

**Comment by ryzokuken** [comment]
@526avijitgupta LGTM, wdyt?

File: index.html (hunk 1 above)
Comment (526avijitgupta) [reply index.html]: Please remove the empty file and it's inclusion from here. No point of keeping it now.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>osdc.github.io</title>
+    <link href="https://fonts.googleapis.com/css?family=Assistant:200|Monoton|Roboto:300" rel="stylesheet">
+    <link rel="stylesheet" href="styles/style.css">
+  </head>
+  <body>
+    <main>
+      <div class="gradientbox">
+          <div class="logo">OSDC</div>
```
Comment (526avijitgupta) [inline index.html]: Incorrect indetation. De-indent by 2 spaces.

**Comment by 526avijitgupta** [comment]
I left a couple of comments - minor code fixes. Otherwise, the page looks nice. Great!

File: index.html (hunk 1 above)
Comment (saranshbarua) [reply index.html]: Removed script.js

**Comment by ryzokuken** [comment]
@saranshbarua why did you unindent everything by 2 spaces? @526avijitgupta asked you to do that for just one line.

**Comment by 526avijitgupta** [comment]
LGTM. Merging !!
Thanks @saranshbarua and @ryzokuken

**Comment by 526avijitgupta** [comment]
@ryzokuken Yeah, I had asked for one line only. But the current indentation seems fine as well. wdyt ?

**Comment by ryzokuken** [comment]
@526avijitgupta yeah, it seems fine.

**Pull request merged**